react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -172,15 +173,49 @@ function onClick(event) {
 
 const styles = theme => {
     return {
+        actions: {
+            margin: '10px auto',
+            textAlign: 'center'
+        },
+        connectionStatsTable: {
+            '&, & > table': {
+                fontSize: '12px',
+                fontWeight: '400',
+
+                '& td': {
+                    padding: '2px 0'
+                }
+            },
+            '& > table': {
+                whiteSpace: 'nowrap'
+            },
+
+            '& td:nth-child(n-1)': {
+                paddingLeft: '5px'
+            },
+
+            '& $upload, & $download': {
+                marginRight: '2px'
+            }
+        },
         contextMenu: {
             position: 'relative',
             marginTop: 0,
             right: 'auto',
             padding: `${theme.spacing(2)}px ${theme.spacing(1)}px`,
-            marginLeft: '4px',
-            marginRight: '4px',
-            marginBottom: '4px'
-        }
+            marginLeft: theme.spacing(1),
+            marginRight: theme.spacing(1),
+            marginBottom: theme.spacing(1)
+        },
+        download: {},
```

Comment:
Please remove all the empty classes

Reply:
It is a pseudo-class. It is used in a selector in `connectionStatsTable` class.

Reply:
Oh, yeah, my bad. I see the upload and download, but not `statsContainer`. Can you remove that please?

Reply:
Oh, yeah, my bad. I see the upload and download, but not `statsContainer`. Can you remove that please?